ccl/changefeedccl: TestChangefeedUpdateProtectedTimestamp failed

ccl/changefeedccl.TestChangefeedUpdateProtectedTimestamp [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20384099?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20384099?buildTab=artifacts#/) on master @ [afdad220750ae62bc9dafbe6ca9d22ab4941095e](https://github.com/cockroachdb/cockroach/commits/afdad220750ae62bc9dafbe6ca9d22ab4941095e):


```
=== RUN   TestChangefeedUpdateProtectedTimestamp
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/0b03c14d06717b86c07a46716d121fa1/logTestChangefeedUpdateProtectedTimestamp1402323321
    test_log_scope.go:82: use -show-logs to present logs inline
=== RUN   TestChangefeedUpdateProtectedTimestamp/kafka
    helpers_test.go:1393: making server as system tenant
    helpers_test.go:1469: making kafka feed factory
panic: Counters should not decrease, prev: 1004890000000, new: -4293759806000000.

goroutine 88162 [running]:
github.com/cockroachdb/cockroach/pkg/util/metric.(*Counter).Update(0xc007c9b720, 0xfff0bed924033480)
	pkg/util/metric/metric.go:818 +0x106
github.com/cockroachdb/cockroach/pkg/server/status.(*RuntimeStatSampler).SampleEnvironment(0xc00a92c908, {0x89233e88, 0xc0064f0460}, 0xc0024da760)
	runtime.go:1054 +0x862
github.com/cockroachdb/cockroach/pkg/server.startSampleEnvironment.func1({0x89233e88, 0xc0064f0460})
	pkg/server/env_sampler.go:186 +0x1a6
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func1({0x89233e88, 0xc0064f0460})
	pkg/util/stop/stopper.go:438 +0xac
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 87409
	pkg/util/stop/stopper.go:436 +0x16c
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedUpdateProtectedTimestamp.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/changefeedccl/protected_timestamps_test.go
+++ b/pkg/ccl/changefeedccl/protected_timestamps_test.go
@@ -154,7 +154,8 @@ func TestChangefeedUpdateProtectedTimestamp(t *testing.T) {
 		}
 	}
 
-	cdcTestWithSystem(t, testFn, feedTestEnterpriseSinks)
+	// Skip kafka sink because it's not compatible with s390x/IBM architecture
+	cdcTestWithSystem(t, testFn, feedTestEnterpriseSinks, feedTestOmitSinks("kafka"))
 }
 
 // TestChangefeedProtectedTimestamps asserts the state of changefeed PTS records

